Packs/Bitbucket/Integrations/Bitbucket/Bitbucket.py

```diff
@@ -0,0 +1,1481 @@
+                "name": destination_branch
+            }
+        }
+    if reviewer_id:
+        reviewers_arr = reviewer_id.split(',')
+        account_id_list = []
+        for id in reviewers_arr:
+            dict = {
+                "account_id": id
+            }
+            account_id_list.append(dict)
+        body["reviewers"] = account_id_list
+    if description:
+        body["description"] = description
+    if close_source_branch:
+        body["close_source_branch"] = argToBoolean(close_source_branch)
+    return body
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def test_module(client: Client) -> str:
+    """ Test the connection to bitbucket.
+    Args:
+        client: A Bitbucket client.
+    Returns:
+        'ok' if the connection was successful, else throws exception.
+    """
+    params = {'pagelen': 1}
+    try:
+        client.get_project_list_request(params=params)
+        return "ok"
+    except Exception:
+        raise Exception('There was a problem in the authentication process.')
+
+
+def project_list_command(client: Client, args: Dict) -> CommandResults:
+    """ Returns a list of all the projects in Bitbucket.
+    Args:
+        client: A Bitbucket client.
+        args: Demisto args.
+    Returns:
+        A CommandResult object with a success message, in case of a successful action.
+    """
+    limit = int(args.get('limit', 50))
+    project_key = args.get('project_key')
+    page = int(args.get('page', 1))
+    check_args(limit, page)
+    page_size = min(100, limit)
+    params = {
+        'page': page,
+        'pagelen': page_size
+    }
+    response = client.get_project_list_request(params, project_key)
+
+    if project_key:
+        results = [response]
+        readable_name = f'The information about project {project_key.upper()}'
+    else:
+        results = check_pagination(client, response, limit)
+        readable_name = f'List of projects in {client.workspace}'
+
+    human_readable = []
+
+    for value in results:
+        d = {'Key': value.get('key'),
+             'Name': value.get('name'),
+             'Description': value.get('description'),
+             'IsPrivate': value.get('is_private')}
+        human_readable.append(d)
+
+    headers = ['Key', 'Name', 'Description', 'IsPrivate']
+
+    readable_output = tableToMarkdown(
+        name=readable_name,
+        t=human_readable,
+        removeNull=True,
+        headers=headers
+    )
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix='Bitbucket.Project',
+        outputs=results,
+        raw_response=results
+    )
+
+
+def open_branch_list_command(client: Client, args: Dict) -> CommandResults:
+    """ Returns a list of all the open branches in Bitbucket.
+    Args:
+        client: A Bitbucket client.
+        args: Demisto args.
+    Returns:
+        A CommandResult object with the list of the branches.
+    """
+    limit = int(args.get('limit', 50))
+    repo = args.get('repo')
```

Comment:
```suggestion
    repo = args.get('repo', client.repository)
```
Do this in all commands and remove the check for the repo